src/com/oltpbenchmark/benchmarks/tpcc/procedures/NewOrder.java

```diff
@@ -182,7 +182,7 @@ public NewOrder() {
       } else {
         sb.append(",?");
       }
-      stmtGetStockSQLArr[i - 1] = new InstrumentedSQLStmt(stmtGetStockSQL.getHistogram(), sb.toString() + ")");
+      stmtGetStockSQLArr[i - 1] = new InstrumentedSQLStmt(stmtGetStockSQL.getHistogram(), sb.toString() + ") FOR KEY SHARE");
```

Comment:
we should add this clause in the prepared statement, not here, if possible

do you know why we were adding the right hand parens?

Reply:
@robertsami could you please clarify the question?

the `stmtGetStockSQLArr` array stores queries which are used to build prepared statements later at line https://github.com/yugabyte/tpcc/blob/master/src/com/oltpbenchmark/benchmarks/tpcc/procedures/NewOrder.java#L410